nilearn/plotting/surf_plotting.py

```diff
@@ -859,7 +859,7 @@ def plot_img_on_surf(stat_map, surf_mesh='fsaverage5', mask_img=None,
         cbar_ax = fig.add_subplot(32, 1, 32)
         fig.colorbar(sm, cax=cbar_ax, orientation='horizontal')
 
-    fig.subplots_adjust(wspace=-0.02, hspace=0.0)
+    fig.subplots_adjust(wspace=-0.02, hspace=0)
```

Comment:
Is there a reason you changed this to an integer?

Reply:
That was a mistake, I removed this change in my new commit.